db/migrate/20201105090756_create_texts.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTexts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :texts do |t|
+      t.string :genre
+      t.string :title
+      t.string :content
```

Comment:
`content` はかなりの長文が入る可能性があるので `string` ではなく `text` にしておきましょう。

```suggestion
      t.text :content
```

変更後に `rails db:migrate:reset` を実行して下さい。

（本来はロールバックしてマイグレーションファイルを修正してマイグレーションを実行しますが，今回はテーブルが他にないので上記でOK）